### ISSUE TITLE
Support for Cocoapods

### ISSUE THREAD

**Issue by honkmaster** [an0/WLActivityViewController#1]
Please add this component to cocoapods.

**Comment by honkmaster** [comment]
Is this component still maintained? Otherwise, I will create a fork and publish to Cocoapods.

**Comment by an0** [comment]
Yes, I'm still using it. I just don't use Cocoapods. But you are welcome to submit PR.

**Comment by honkmaster** [comment]
Of course I can do it. however, you still have to submit the podspec to the cocoapods repo. This, without your consent this is useless.

**Comment by an0** [comment]
You have my consent to submit it.
